docs/modules/ROOT/pages/how-tos/metadata/sboms.adoc

```diff
@@ -1 +1,152 @@
-= Inspecting SBOMs
\ No newline at end of file
+= Inspecting SBOMs
+
+== Software bill of materials
+A software bill of materials (SBOM) provides greater transparency for your software supply chain. {ProductName} provides link:https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf[build SBOMs], which list all the software libraries that a component uses. Those libraries can enable specific functionality or facilitate development. 
+
+You can use an SBOM to better understand the composition of your software. In particular, you can analyze your SBOM to determine if your application has any vulnerable dependencies. Furthermore, to comply with cybersecurity regulations, you might need to provide your customers with the SBOM for your application's components. For example, in the United States, the link:https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/[Executive Order on Improving the Nation's Cybersecurity] requires vendors to provide SBOMs, and in Europe, the link:https://www.cisa.gov/sites/default/files/2023-09/EU%20Commission%20SBOM%20Work_508c.pdf[Cyber Resilience Act] does the same.
```

Comment:
> you can analyze your SBOM to determine if your application has any vulnerable dependencies

The SBOM doesn't have any vulnerability information. In order to determine this, you would have to collate the data in the SBOM with some other data source (i.e. a vulnerability database).

Reply:
That's exactly what I'm trying to communicate with "analyze your SBOM." For example, with RHTAP, we say you "analyze your SBOM with TPA" to get actionable info about vulns. 
Would this work?
 "In particular, you can use various tools to scan your SBOM and determine if any of your dependencies have known vulnerabilities."

Reply:
that seems reasonable.